<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>테이블 선택</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>광산구 전체 인구</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>부동산업체/산업단지</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>편의 시설</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>음식 시설</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>혼합 시설</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="all_people_graph"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
